src/tlo/methods/tb.py

```diff
@@ -1933,6 +1933,74 @@ def apply(self, person_id, squeeze_factor):
             return ACTUAL_APPT_FOOTPRINT
 
 
+class HSI_Tb_ClinicalDiagnosis(HSI_Event, IndividualScopeEventMixin):
+    """
+    This is a clinical diagnosis appt which is called when other tests have not been
+    available and only a clinical diagnosis is required
+
+    * it does not include any of the routine tests for TB or HIV
+    therefore property tb_date_tested is not updated
+    * It only requires 0.5 footprint of Under5OPD since it will almost exclusively
+    be used for children unable to get xrays following initial diagnostic consultations
+    """
+
+    def __init__(self, module, person_id, suppress_footprint=False):
+        super().__init__(module, person_id=person_id)
+        assert isinstance(module, Tb)
+
+        assert isinstance(suppress_footprint, bool)
+        self.suppress_footprint = suppress_footprint
+
+        self.TREATMENT_ID = "Tb_Test_Clinical"
+        self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({"Under5OPD": 0.5})
+        self.ACCEPTED_FACILITY_LEVEL = '1a'
+
+    def apply(self, person_id, squeeze_factor):
+        """ Do the screening and referring process """
+
+        df = self.sim.population.props
+        now = self.sim.date
+        person = df.loc[person_id]
+        test_result = None
+
+        # If the person is dead or already diagnosed, do nothing do not occupy any resources
+        if not person["is_alive"] or person["tb_diagnosed"]:
```

Comment:
The none / all logic could be accomplished more efficiently used sets, I think:

i.e. 
```python
if not any(x in self.module.symptom_list for x in persons_symptoms)
```
and 
```python
if all(x in self.module.symptom_list for x in persons_symptoms)
```

becomes:
```python
set_of_symptoms_that_indicate_tb = set(self.module.symptom_list)
if not set_of_symptoms_that_indicate_tb.intersect(persons_symptoms):
```
and 
```python
if set_of_symptoms_that_indicate_tb.issubset(persons_symptoms):
```